Assets/MixedRealityToolkit/Interfaces/InputSystem/Handlers/IMixedRealityPointerUpdatedHandler.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine.EventSystems;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Interface to implement to react to simple pointer input and pointer updates.
+    /// </summary>
+    public interface IMixedRealityPointerUpdatedHandler : IMixedRealityPointerHandler
```

Comment:
I like OnPointerDragged instead of OnPointerUpdated as well. So IMixedRealityPointerDraggedHandler, OnPointerDragged.